nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-avro-record-utils/src/main/java/org/apache/nifi/schema/access/SchemaAccessUtils.java

```diff
@@ -50,7 +50,7 @@
         "The content of the FlowFile contains a reference to a schema in the Schema Registry service. The reference is encoded as a single "
             + "'Magic Byte' followed by 4 bytes representing the identifier of the schema, as outlined at http://docs.confluent.io/current/schema-registry/docs/serializer-formatter.html. "
             + "This is based on version 3.2.x of the Confluent Schema Registry.");
-
+    public static final AllowableValue INFER_SCHEMA = new AllowableValue("infer", "Infer from JSON");
```

Comment:
Shouldn't this be "Infer from Result" or something? It could be used by other processors to infer the schema from whatever object is returned.

Reply:
Probably. I'll go ahead and make that change.